bookbook/Views/BookTextView.swift

```diff
@@ -27,19 +27,48 @@ extension ViewController {
     summaryTitle.text = "Summary"
     summaryTitle.font = UIFont.systemFont(ofSize: 18, weight: .bold)
     // 요약 속성
-    summary.text = books[index].summary
+//    print(books[index].summary.count)
+    summaryUpdate()
+
+    summaryButton.setTitle(isExpanded ? "접기" : "더 보기", for: .normal)
+    summaryButton.setTitleColor(.systemBlue, for: .normal)
+    summaryButton.titleLabel?.font = UIFont.systemFont(ofSize: 14)
+  }
+
+  func summaryUpdate() {
+    let isExpanded = UserDefaults.standard.bool(forKey: isExpandedKey(for: index))
+    let summaryText = books[index].summary
+
+    if summaryText.count >= 450 {
+      summaryButton.isHidden = false
+      if !isExpanded {
+        summary.text = summaryText.prefix(450) + "..."
+        summaryButton.setTitle("더 보기", for: .normal)
+      } else {
+        summary.text = summaryText
+        summaryButton.setTitle("접기", for: .normal)
+      }
+    } else if summaryText.count < 450 {
+      summaryButton.isHidden = true
+      summary.text = summaryText
```

Comment:
조금 중복되는 느낌이 있어서 삼항 연산자를 써서 줄여봤습니다. 이렇게 하는 방법도 좋을 것 같습니다 

```suggestion
    if summaryText.count >= 450 {
      summaryButton.isHidden = false

      summary.text = isExpanded ? summaryText : summaryText.prefix(450) + "..."
      summaryButton.setTitle(isExpanded ? "접기" : "더 보기", for: .normal)

    } else { // summaryText.count < 450
      summaryButton.isHidden = true
      summary.text = summaryText
    }
```

Reply:
오, 고맙습니다! 그러네요 굳이 elif 남발할 필요가 없겠네요